NewStarterProject/Controllers/DoorController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NewStarterProject.Dtos;
+using NewStarterProject.Model;
+using NewStarterProject.Services;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace NewStarterProject.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DoorController : ControllerBase
```

Comment:
Again, I really like that you are keeping controllers light weight. They are simply entry points and return a result, no application logic here and that is great to see!